### PR TITLE
Include cheapest price using text placeholder

### PR DESCRIPTION
**Pull request by nglgzz**
This should work on all text, not only on `mainActionButtonText`.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -212,57 +220,67 @@ export default class Index extends React.Component<Props, State> {
...
+                    isMobile={isMobile}
+                    cityTag={cityTag}
+                    isStopover={cityData.isStopover}
+                    usedLocales={usedLocales}
+                  />
+                </FetchedProvider>
+                <Hero
+                  logo={cityData.cityLogo}
+                  photo={cityData.mainPhoto}
+                  departureForLowestPrice={cityData.departureForLowestPrice}
```
Comment (alexedev) [inline pages/index.js]: why do we need these props in Hero?

File: etc/fetchLowestPrice.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const data = (await response.json()).data.get_flights;
+  const price = {
+    amount: data.bestResults[0]?.price,
+    currency: data.currency,
+  };
+
+  if (price.amount && price.currency) {
+    return `${price.amount} ${price.currency}`;
+  }
+  return "-";
```
[the remark this answers is not in the document]
Comment (nglgzz) [reply etc/fetchLowestPrice.js]: You're right, what should I put in case no results are found?

File: pages/index.js (hunk 1 above)
Comment (nglgzz) [reply pages/index.js]: We don't, I forgot to remove them, will do that now.

File: etc/fetchLowestPrice.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexedev) [reply etc/fetchLowestPrice.js]: I think we should throw an error and fail the build

File: etc/fetchLowestPrice.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  const now = DateTime.local();
+  const dateFrom = now.toFormat("dd/MM/yyyy");
+  const dateTo = now.plus({ months: 3 }).toFormat("dd/MM/yyyy");
+
+  const variables = {
+    parameters: {
+      dateFrom,
+      dateTo,
+      to: arrival,
+      flyFrom: departure === "EU" ? "europe" : departure,
```
[the remark this answers is not in the document]
Comment (nglgzz) [reply etc/fetchLowestPrice.js]: It does. Apparently the API doesn't like EU and works only with `europe`, US is fine. I agree it looks strange though. I'm just not sure what would be a better way to do it. Should we change the value on DatoCMS instead?

File: etc/fetchLowestPrice.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nglgzz) [reply etc/fetchLowestPrice.js]: Sure, makes sense. Will do that.

File: etc/fetchLowestPrice.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (alexedev) [reply etc/fetchLowestPrice.js]: no, It's ok. maybe just put the departure type as "EU" | "US", not just a string
and having a comment why we need to convert "EU" to "europe".
Again, will it work if departure is null/undefined?

**Pull request merged**